integration-test-groups/azure/azure-resources.sh

```diff
@@ -69,6 +70,14 @@ function deleteResources() {
     az group delete --name ${RESOURCE_GROUP} --yes
 }
 
+if ! which az > /dev/null; then
```

Comment:
Great idea to check. Does which output on stderr in this case ? Maybe "2>" could be tried.

Reply:
I'll do that so that all output is dumped to `/dev/null`.